package_manifest.yaml

```diff
@@ -263,7 +263,10 @@ katello_packages:
     rubygem-hammer_cli_foreman_virt_who_configure: {}
     rubygem-hammer_cli_katello: {}
     rubygem-jenkins_api_client: {}
-    rubygem-katello: {}
+    rubygem-katello:
+      build_package_tito_releaser_args:
```

Comment:
One more thing (thanks for being the guinea pig), add this under `rubygem-katello`:

```
releasers:
     - koji-katello-jenkins
```